test.sh

```diff
@@ -59,11 +59,12 @@ function usage() {
    echo "To run CI-regression tests       : INCLUDE_SLOW_TESTS=true ./${Me}"
    echo "To run nightly regression tests  : RUN_NIGHTLY_TESTS=true ./${Me}"
    echo "To run make build-and-test tests : RUN_MAKE_TESTS=true ./${Me}"
-   echo
+   echo " "
```

Comment:
In CI stdout, we seem to be stripping out these empty blank lines. Insert a `" " ` so blank lines are seen in CI outputs, too.